sql/moz-fx-data-shared-prod/firefoxdotcom_derived/ga_clients_v1/bigconfig.yml

```diff
@@ -0,0 +1,18 @@
+type: BIGCONFIG_FILE
```

Comment:
We also need to add monitoring configuration to the metadata.yaml file. Here's an example:
https://github.com/mozilla/bigquery-etl/blob/7671fbc85d2b968e43efb58b5935549a2a6bf413/sql/moz-fx-data-shared-prod/telemetry_derived/ssl_ratios_v1/metadata.yaml#L17-L18